src/task.py

```diff
@@ -1,6 +1,7 @@
 import argparse
 import ee
 from task_base import SCLTask
+from reclass_list import lc_elev_reclass_esa
```

Comment:
`lc_elev_reclass_esa` is a list... I suggest naming the module something more generic (instead of `reclass_list`)... maybe `landcover` (landcover.py) for future additions related to LULC preprocessing as opposed to vegetation height, or `parameters` for future addtional params that we don't want to put in env or commandline args?